tests/default_tasks_test/arm_test.py

```diff
@@ -0,0 +1,106 @@
+"""Test default rastrigin using MAP Elites"""
+
```

Comment:
This file looks more like testing MAP-Elites rather than arm task. Maybe we could think about just a few simple tests that check the implementation works as expected.